test/views/pageNav.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
```

Comment:
I know this is probably just a test stub, but it's confusing that it's a copy of the loadTemplate test. Can we remove it and just bring it in if and when we actually have a test for the view?